pkg/controller/base_controller.go

```diff
@@ -92,7 +99,15 @@ func (c *controller) Run(ctx context.Context) error {
 	wg.Add(c.numWorker)
 	for i := 0; i < c.numWorker; i++ {
 		go func() {
-			wait.Until(c.runWorker, c.syncPeriod, ctx.Done())
+			wait.Until(c.runWorker, time.Second, ctx.Done())
```

Comment:
assuming here c.runWorker will be called every second, duration seems to be very aggressive.. we can increase the time..
same applies for below watigroup also

Reply:
ok. I'll set it to 10 seconds.